app/controllers/search_controller.rb

```diff
@@ -9,37 +9,38 @@ def explore
 
   def search
   	sort_arr = sort_order
-  	@repositories = Repository.search do
-	    keywords params[:q] 
+  	@repositories = Repository.where("title LIKE ?
+                                  OR description LIKE ?
+                                  OR user_username LIKE ?
+                                  OR category LIKE ?",
+                                  "%#{params[:q]}%", "%#{params[:q]}%", "%#{params[:q]}%", "%#{params[:q]}%").paginate(:per_page=>12,:page=>params[:page]) do
```

Comment:
Do you need the `paginate` params on the end of this? It looks like those are added the query inside the block that follows (line 18). It feels redundant to have both